components/list/list-item-mixin.js

```diff
@@ -351,6 +376,9 @@ export const ListItemMixin = superclass => class extends ListItemDragDropMixin(L
 
 	_onFocusIn() {
 		this._focusing = true;
+		if (this.role !== 'rowgroup' || !tabPressed || hasDisplayedKeyboardTooltip) return;
```

Comment:
Only show the tooltip if the list is in `grid` mode, and focus is the result of tabbing into the list, and the tooltip has not already been displayed.